Makefile

```diff
@@ -12,8 +12,14 @@ OUTPUT_TAGS := $(STATUS_TAGS)ietf-cellar-tags-$(VERSION_TAGS)
 
 all: $(OUTPUT_MATROSKA).html $(OUTPUT_MATROSKA).txt $(OUTPUT_MATROSKA).xml $(OUTPUT_CODEC).html $(OUTPUT_CODEC).txt $(OUTPUT_CODEC).xml $(OUTPUT_TAGS).html $(OUTPUT_TAGS).txt $(OUTPUT_TAGS).xml
 
-ebml_matroska_elements4rfc.md: ebml_matroska.xml transforms/ebml_schema2markdown4rfc.xsl
-	xsltproc transforms/ebml_schema2markdown4rfc.xsl ebml_matroska.xml > $@
+matroska_xsd.xml: transforms/schema_clean.xsl ebml_matroska.xml
+	xsltproc transforms/schema_clean.xsl ebml_matroska.xml > $@
+
+check: matroska_xsd.xml
+	xmllint --noout --schema ../ebml-specification/EBMLSchema.xsd matroska_xsd.xml
```

Comment:
This is a new thing and requires the EBML specification repo to be downloaded in a certain place. 
This could be used to validate changes to the Matroska Schema on Github before allowing a push.

Reply:
Seems reasonable to me.